javascript/javascript2/week3/giphyApi/style.css

```diff
@@ -0,0 +1,45 @@
+main {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    background: azure;
+}
+h1 {
```

Comment:
You could avoid this tag styling, rather add some class for the h1's with these styles.